app/grandchallenge/components/models.py

```diff
@@ -1676,15 +1655,17 @@ class ComponentJob(FieldChangeMixin, UUIDModel):
     runtime_metrics = models.JSONField(default=dict, editable=False)
     error_message = models.CharField(max_length=1024, default="")
     detailed_error_message = models.JSONField(blank=True, default=dict)
-    started_at = models.DateTimeField(null=True)
-    completed_at = models.DateTimeField(null=True)
-    compute_cost_euro_millicents = models.PositiveIntegerField(
-        # We store euro here as the costs were incurred at a time when
-        # the exchange rate may have been different
-        editable=False,
-        null=True,
-        default=None,
-        help_text="The total compute cost for this job in Euro Cents, including Tax",
+    started_at = deprecate_field(models.DateTimeField(null=True))
+    completed_at = deprecate_field(models.DateTimeField(null=True))
+    compute_cost_euro_millicents = deprecate_field(
```

Comment:
The fields cannot be deprecated before the data migration is run otherwise the values will always return None, see https://pypi.org/project/django-deprecate-fields/